modules/serverless-rn-1-28-0.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-28-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.28
+
+{ServerlessProductName} 1.28 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-28-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.7.
+* {ServerlessProductName} now uses Knative Eventing 1.7.
+* {ServerlessProductName} now uses Kourier 1.7.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.7.
+* {ServerlessProductName} now uses Knative Kafka 1.7.
+* The `kn func` CLI plug-in now uses `func` FIXME version.
+
+* Node.js and TypeScript runtimes for {ServerlessProductName} Functions are now Generally Available (GA).
+
+* Multi-container support for Knative Serving is now available as a Technology Preview. This feature allows you to use a single Knative service to deploy a multi-container pod.
+
+* In a future release, the following components of Knative Eventing will be scaled down from two pods to one:
```

Comment:
By "future release" you mean 1.29? I'm curious if this is just a warning for the future release.

Reply:
Yeah, it is indeed just a warning for a future release. I've made it more explicit.